app/models/user.rb

```diff
@@ -36,4 +37,14 @@ def current_friend_requests
     end
     requests
   end
+
+  def books
+    user_books.map do |user_book|
+      create_book(user_book)
+    end
+  end
+
+  def create_book(user_book)
+    BookFacade.find_by_isbn(user_book.isbn)
```

Comment:
When calling on the BookFacade here, the api is getting called everytime. We should add some logic in the facade or the service class to check our DB prior to making a call to the service

Reply:
Cool, I can add this in my service class for my Book show branch.